service/bin/d-installer

```diff
@@ -43,7 +43,7 @@ def start_service(name)
   service_runner.run
 end
 
-ORDERED_SERVICES = [:language, :software, :users, :manager].freeze
+ORDERED_SERVICES = [:questions, :language, :software, :storage, :users, :manager].freeze
```

Comment:
when I see it question is what is future of that containerized dinstaller? if we want to keep container per process we need to update way how it is run. ( but as we do not have yet docker-compose or something similar, probably not big issue )

Reply:
Yes, we need to think a little bit about it. Actually, the questions service does not even depend on YaST at all.